kubetest/aksengine.go

```diff
@@ -1303,6 +1304,8 @@ func (c *aksEngineDeployer) BuildTester(o *e2e.BuildTesterOptions) (e2e.Tester,
 		csiDriverName = "azurefile-csi-driver"
 	} else if *testBlobfuseCSIDriver {
 		csiDriverName = "blobfuse-csi-driver"
+	} else if *testSecretStoreCSIDriver {
+		csiDriverName = "secrets-store-csi-driver"
```

Comment:
Note that all CSI drivers call `make e2e-test` in `$(GOPATH)/src/sigs.k8s.io/${csiDriverName}` to kick off the e2e tests.
LGTM otherwise.

Reply:
That's right. I'm going to be add a rule `e2e-test` in the makefile which will install the dependencies and kick off the tests.